src/Propel/Generator/Platform/MysqlPlatform.php

```diff
@@ -366,6 +366,18 @@ public function getColumnDDL(Column $col)
             $ddl[] = $sqlType;
         }
         $colinfo = $col->getVendorInfoForType($this->getDatabaseType());
+        if ($colinfo->hasParameter('Unsigned')) {
+            $unsigned = $colinfo->getParameter('Unsigned');
+            switch (strtoupper($unsigned)) {
+                case 'FALSE': break;
+                case 'TRUE':
+                    $ddl[] = 'UNSIGNED';
+                    break;
+                default:
+                    throw new EngineException('Unexpected value "'.$unsigned.'" for MySQL vendor column parameter "Unsigned", expecting "true" or "false".');
+            }
+            unset($unsigned);
```

Comment:
Yeah sorry, I have the habit of unsetting as soon as I am done with a local var so my IDE stops proposing it to me when autocompleting outside of the codeblock I'm using it.
Would you like me to update the PR without this line?